commit dc70bbc3bf9e9a2cb02ed7319409ecad1e034ec4 (refs/changes/83/5583/2)
Author: Robert Bogart <robert.bogart@joyent.com>
Date:   2019-02-15T07:07:59+00:00 (8 months ago)
    
    MANATEE-343 manatee full rebuilds should pull from the primary where possible
    MANATEE-375 Update dev zone scripts for MANATEE-335
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Dave Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index d90f925..53c1c51 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+https://github.com/joyent/manatee.git#179a24faf5fac6c477d0459646c78c1fccf4fce8",
+        "manatee": "git+https://github.com/joyent/manatee.git#35be0bca5871d9ba35f68c75b0ec1e8aa8538f42",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
